demos/bombs/bombs.cpp

```diff
@@ -196,9 +197,11 @@ void BombsFrame::OnEasyCorner(wxCommandEvent& WXUNUSED(event))
                this
              );
 
-    if(ok!=wxYES)return;
+    if(ok!=wxYES)return GetMenuBar()->Check(bombsID_EASYCORNER, m_easyCorner);
 
     m_easyCorner = !m_easyCorner;
+    
+    GetMenuBar()->Check(bombsID_EASYCORNER, m_easyCorner);
```

Comment:
I don't think this is necessary, the check menu item should have been already toggled, no?

Reply:
Yes, it's been removed now.